main.go

```diff
@@ -384,7 +388,12 @@ func makeServiceChanges(cf *Cloudflare, kubeClientset *kubernetes.Clientset, ser
 		service.Annotations[annotationCloudflareState] = string(cloudflareStateByteArray)
 
 		// update service, because the state annotations have changed
-		service, err = kubeClientset.CoreV1().Services(service.Namespace).Update(service)
+		service, err = kubeClientset.CoreV1().Services(service.Namespace).Update(ctx, service, metav1.UpdateOptions{
+			TypeMeta:        metav1.TypeMeta{},
+			DryRun:          []string{},
+			FieldManager:    "",
+			FieldValidation: "",
```

Comment:
I think the default value has to be `Ignore`, `warn` or `strict`. If its default then probably not defining this is a good idea

Reply:
Removed.